app/Models/PriceHistory.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class PriceHistory extends Model
+{
+    use HasFactory;
+    const UPDATED_AT = null;
+
+    protected $casts = [
+        'price' => 'decimal:2'
```

Comment:
Co to daje? to :2?

Reply:
Do ilu liczb po przecinku ma być zaokrąglona liczba. Potrzebne mi to było, bo gdzieś w testach 2 teoretycznie takie same floaty nie były sobie równe :/

Reply:
Matematyka według PHP :)